src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -202,14 +202,28 @@ export class NotebookEditor extends EditorPane implements IFindNotebookControlle
 			container.style.height = '100%';
 			this._notebookContainer = DOM.append(parentElement, container);
 			input.container = this._notebookContainer;
+			this.findSplitView(parentElement);
 			return Promise.resolve(this.bootstrapAngular(input));
 		} else {
 			this._notebookContainer = DOM.append(parentElement, input.container);
+			this.findSplitView(parentElement);
 			input.doChangeLayout();
 			return Promise.resolve(null);
 		}
 	}
 
+	private findSplitView(parentElement: HTMLElement): void {
```

Comment:
If anyone has any suggestions for this, would be great. Without this, no scrollbar appears unfortunately 😢

Reply:
Could you explain more what this is doing? (something like this should have that in comments anyways)

It definitely seems fragile to have to do this - especially since you're checking that the class name is exactly `split-view-view visible`. What if another class is added to the split view?

But let me see if I get this right - so the diff editor is basically just 2 NotebookEditors within a split view? How do normal text/code editors handle this so that the scrollbars appear correctly?